pkg/driver/manager.go

```diff
@@ -27,6 +27,13 @@ func FilterID(id string) FilterFn {
 	}
 }
 
+// FilterDeviceType returns a filter function to get registered drivers which matches t type
+func FilterDeviceType(t DeviceType) FilterFn {
+	return func(d Driver) bool {
+		return d.Info().DeviceType == t
+	}
+}
+
```

Comment:
It maybe nice to have
```go
// FilterAnd returns a filter function to take logical conjunction of given filters.
func FilterAnd(filters... FilterFn) FilterFn {
	return func(d Driver) bool {
		for _, f := range filters {
			if !f(d) {
				return false
			}
		}
		return true
	}
}
```

Reply:
Oh yes, this looks really good! Feel free to add it. This will make the code cleaner for sure.

Reply:
I'll add it!

Reply:
Added